scripts/publish-staged

```diff
@@ -21,80 +21,106 @@ IMAGE_NAME="amazon/amazon-ecs-agent"
 
 AWS_PROFILE=""
 STAGE_S3_BUCKET=""
-PUBLISH_S3_BUCKET="amazon-ecs-agent"
```

Comment:
We still need to export images and push them to S3.  We need this as the source of truth for ecs-init, for our AMI builds, and as we don't want to rely on Docker Hub in the critical path.

Reply:
Let's chat a bit offline so I can understand this change a bit more.